models/src/anemoi/models/layers/chunk.py

```diff
@@ -189,18 +181,17 @@ def __init__(
                 out_features=num_channels,
                 layer_kernels=layer_kernels,
                 n_extra_layers=mlp_extra_layers,
-                activation=activation,
             )
         else:
             self.emb_edges = None
 
         self.build_blocks(
             GraphConvProcessorBlock,
-            num_channels,
-            num_channels,
+            in_channels=num_channels,
+            out_channels=num_channels,
+            num_chunks=1,
```

Comment:
where is this `num_chunks=1` coming from? (asking also since the default before seemed `num_chunks: int = 2`)

Reply:
The default before is defined here: https://github.com/ecmwf/anemoi-core/blob/07bd6183dc835a1132faa3af6ff9d0b5cd75477c/models/src/anemoi/models/layers/block.py#L282 which is `1`.

As it was never passed it used the default kwarg defined in the thing. This may also be confusing as num_chunks in the mapper is different that num_chunks in the blocks if I recall correctly.